pkg/reconciler/events/scheduler/scheduler.go

```diff
@@ -200,14 +200,18 @@ func (r *Reconciler) reconcileJob(ctx context.Context, scheduler *v1alpha1.Sched
 		} else if st.Code() == codes.NotFound {
 			// Create the job as it does not exist. For creation, we need a parent, extract it from the jobName.
 			parent := resources.ExtractParentName(jobName)
+			customAttributes := make(map[string]string)
+			// Add our own jobName here...
+			customAttributes[v1alpha1.JobName] = jobName
```

Comment:
this custom attribute will come in the PubSub message? If so, then we can just add an attribute to identify the converter, as Storage does... Otherwise, in the future, we might have a problem if we want to have a single receive adapter for multiple sources...

Reply:
do you know if we can do that with CAL as well?

Reply:
Yep. The custom attributes will come in the PubSub message, We can add an attribute to identify the converter as Storage does. However, we will also need an attribute to identify the cloud scheduler job.

Reply:
For CAL, currently I don't find any way to set up custom attributes.

Reply:
OK, we can leave CAL as is, but if we can set custom attributes here, then let's add it... both the job and the "adapter" type...